src/__swaps__/screens/Swap/hooks/useAnimatedSwapStyles.ts

```diff
@@ -203,12 +203,18 @@ export function useAnimatedSwapStyles({
         SPRING_CONFIGS.springConfig
       ),
       borderRadius: withSpring(isBottomSheetOpen ? 40 : 0, SPRING_CONFIGS.springConfig),
-      bottom: withSpring(isBottomSheetOpen ? Math.max(safeAreaInsetValues.bottom, 28) : -2, SPRING_CONFIGS.springConfig),
+      bottom: IS_TEST
```

Comment:
a lot of this conditional styling for IS_TEST is necessary for maestro to see elements